### PR TITLE
Change links and text to the current repository name

### LINKED ISSUE #120: Change links and text to the current repository name
## Description
As a contributor,
I need all the links and text that is using `open-source-programs-web` changed into `anitab-forms-web`,
so that there is no confusion and 404 pages for newcomers.

### Update [Required]
- [ ] Change `open-source-programs-backend` to `anitab-forms-backend`.

## Definition of Done
- [ ] All of the required items are completed.
- [ ] Approval by 1 mentor.

## Estimation
3 hours.

### PR DESCRIPTION
**Pull request by keshavagarwal17**
### Description

I changed all the texts and link which includes `open-source-programs-web` and `open-source-programs-backend ` to `anitab-forms-web` and `anitab-forms-backend` respectively.

Fixes #120 

### Type of Change:

- Documentation

### Checklist:

- [x] My PR follows the style guidelines of this project
- [x] I have performed a self-review of my own code or materials

**Code/Quality Assurance Only**
- [x] My changes generate no new warnings

### CODE REVIEW

**Review by keshakaneria — CHANGES_REQUESTED** [review]
The following places still needs to be updated:
- `docs/README.md`
- Check all `*.js` files which currently aren't edited. We will have to replace them too even if the files aren't seen to users, so that in future there is no confusion of names.

**Comment by keshavagarwal17** [comment]
@keshakaneria I made the changes please review it again.

**Review by keshakaneria — CHANGES_REQUESTED** [review]
You missed [this](https://github.com/anitab-org/anitab-forms-web/blob/develop/docs/src/pages/index.js#L15).

**Comment by keshavagarwal17** [comment]
@keshakaneria should i need to change all `open source programs (osp)` text to `anitab forms program` also? because till now i changed only `open-source-programs-web` and `open-source-programs-backend`.

**Comment by keshakaneria** [comment]
> @keshakaneria should i need to change all `open source programs (osp)` text to `anitab forms program` also? because till now i changed only `open-source-programs-web` and `open-source-programs-backend`.

Oops! My fault, the issue doesn't include this so the changes made are fine 😄

**Review by keshakaneria — DISMISSED** [review]
You are good to go! 🎉😄

File: .github/CONTRIBUTING.md [hunk 1]
```diff
@@ -2,8 +2,8 @@
 
 -   You can join us on [AnitaB.org Open Source Zulip](https://anitab-org.zulipchat.com/). Each active repo has its own stream to direct questions to (for example #powerup or #portal). Open Source Programs stream is [#open-source-progs](https://anitab-org.zulipchat.com/#streams/237907/open-source-progs).
```
Comment (ashwinijha6) [inline .github/CONTRIBUTING.md]: @The-Keshav-Agarwal  just change the the link & the name here also for zulip
Link : https://anitab-org.zulipchat.com/#narrow/stream/237907-anitab-forms
Name of the Stream : [#anitab-forms]

**Review by ashwinijha6 — CHANGES_REQUESTED** [review]

**Review by ashwinijha6 — COMMENTED** [review]
Hey @The-Keshav-Agarwal are you changing the repo name inside the docs/ also for docusaurus ?

**Comment by keshavagarwal17** [comment]
@ashwinijha6 I am changing name in docs folder and in .js file.

File: README.md [hunk 2]
```diff
@@ -1,7 +1,7 @@
 [![project chat](https://img.shields.io/badge/zulip-join_chat-brightgreen.svg)](https://anitab-org.zulipchat.com/#narrow/stream/237907-open-source-progs)
 [![License: GPL v3](https://img.shields.io/badge/License-GPL%20v3-blue.svg)](http://www.gnu.org/licenses/gpl-3.0)
 ![Open Source Love](https://img.shields.io/badge/Open%20%E2%9D%A4%EF%B8%8FSource-blue)
-[![Maintenance](https://img.shields.io/badge/Maintained%3F-yes-green.svg)](https://github.com/anitab-org/open-source-programs-web)
+[![Maintenance](https://img.shields.io/badge/Maintained%3F-yes-green.svg)](https://github.com/anitab-org/anitab-forms-web)
 ![PRs Welcome](https://img.shields.io/badge/PRs-welcome-brightgreen.svg?style=flat-square)
 
 # Open Source Programs (Frontend)
```
Comment (ashwinijha6) [inline README.md:7]: Change the name here to `Anitab Forms Web`

File: README.md [hunk 3]
```diff
@@ -1,7 +1,7 @@
 [![project chat](https://img.shields.io/badge/zulip-join_chat-brightgreen.svg)](https://anitab-org.zulipchat.com/#narrow/stream/237907-open-source-progs)
```
Comment (ashwinijha6) [inline README.md]: The zulip stream link needs to changed here

File: docs/README.md [hunk 4]
```diff
@@ -34,7 +34,7 @@ If you are using GitHub pages for hosting, this command is a convenient way to b
 
 ## Deploying to Surge
 
-Surge is a static web hosting platform. Tt is used to deploy our spen source programs docusaurus from the command line in a minute.
+Surge is a static web hosting platform. It is used to deploy our spen source programs docusaurus from the command line in a minute.
```
Comment (ashwinijha6) [inline docs/README.md]: change it to `spen` to `open`

File: docs/docs/2020-Bismita-Guha.md [hunk 5]
```diff
@@ -4,7 +4,7 @@ title: GSoC-2020 Bismita Guha
 ---
 
 ## Bismita Guha
-I plan to work on the backend and frontend of Open Source Programs in this GSoC period. My mentors are [Maybelline Burgos](https://github.com/mayburgos), [Monal Shadi](https://github.com/Monal5031), [Abha Wadjikar](https://github.com/abha224) and [Siddharth Venu](https://github.com/sidvenu). Checkout the [backend repo](https://github.com/anitab-org/open-source-programs-backend).
+I plan to work on the backend and frontend of Open Source Programs in this GSoC period. My mentors are [Maybelline Burgos](https://github.com/mayburgos), [Monal Shadi](https://github.com/Monal5031), [Abha Wadjikar](https://github.com/abha224) and [Siddharth Venu](https://github.com/sidvenu). Checkout the [backend repo](https://github.com/anitab-org/anitab-forms-backend).
```
Comment (ashwinijha6) [inline docs/docs/2020-Bismita-Guha.md:7]: here also the repo name needs to changed in the first line. What do you think @keshakaneria?
I'm not sure about this as during GSoc'20 the name was `Open Source Programs`

File: docs/docs/Fork,-Clone,-Remote-and-Pull-Request.md [hunk 6]
```diff
@@ -3,40 +3,40 @@ id: Fork,-Clone,-Remote-and-Pull-Request
 title: Fork, Clone, Remote and Pull Request
 ---
 
-To contribute to the project you may work on the [frontend](https://github.com/anitab-org/open-source-programs-web) or [backend](https://github.com/anitab-org/open-source-programs-backend) of the project.
+To contribute to the project you may work on the [frontend](https://github.com/anitab-org/anitab-forms-web) or [backend](https://github.com/anitab-org/anitab-forms-backend) of the project.
 ### Fork the repo
 ![Forking](https://user-images.githubusercontent.com/43119923/81509477-99692080-9328-11ea-89e2-6e54f04b9544.jpg)
```
Comment (ashwinijha6) [inline docs/docs/Fork,-Clone,-Remote-and-Pull-Request.md:8]: Change the screenshot here for fork

File: docs/docs/Getting-Started.md [hunk 7]
```diff
@@ -8,7 +8,7 @@ Hello Newcomer, here are some tips to help you get started contributing to this
 ## Get to know the project
 Here's what you can do to know more about the project:
 
-- Read documentation available on this website and on [README](https://github.com/anitab-org/open-source-programs-web/blob/develop/README.md).
+- Read documentation available on this website and on [README](https://github.com/anitab-org/anitab-forms-web/blob/develop/README.md).
 - You can join the [#open-source-progs](https://anitab-org.zulipchat.com/#narrow/stream/237907-open-source-progs) channel on AnitaB.org Open Source Zulip, to see or participate in the project's discussion.
```
Comment (ashwinijha6) [inline docs/docs/Getting-Started.md]: change the zulip stream link here

**Review by ashwinijha6 — CHANGES_REQUESTED** [review]
@The-Keshav-Agarwal I was just asking because as there are a lot of files in docs/docs where the repo name needs to be changed like:
- Home.md under Home & in the About the Section (2)
- User Interface Design under that change OSP
- Main Base Concepts.md here also it says OSP Application (should this be changed @keshakaneria )
- Future Ideas.md the zulip stream
-  You will need to change the description in index.js

**Comment by ashwinijha6** [comment]
Check with the conflicts

**Comment by codesankalp** [comment]
@The-Keshav-Agarwal Please do the suggested changes.

**Comment by Aaishpra** [comment]
@The-Keshav-Agarwal please update the PR as requested and remove the merge conflicts or this PR will be closed and issue will be made available again.

**Comment by keshavagarwal17** [comment]
@Aaishpra I am closing this PR. please assign this issue to someone else.

**Pull request closed without merging**